Alg_LZW/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OpenFile;
+var LZWOpenFile = new OpenFile.OpenFile();
+if (!(args.Length==2))
```

Comment:
```suggestion
﻿using OpenFile;
var LZWOpenFile = new OpenFile.OpenFile();
if (!(args.Length == 2))
```
в этом файле и дальше много таких замечаний по кодстайлу